README.md

```diff
@@ -115,7 +113,7 @@ To stop the daemon after the build completes, run the following:
 `./gradlew --stop`
 
 ## Running a sample app
-1. To run a sample app on an Android 4.4.4 or later device, plug the device into the computer and ensure that USB debugging is enabled.  Android Studio will recognize the device as long as the device identifies as running a version between Android 10 and Android 19 otherwise Android Studio will run the sample app in the simulator.  Using the simulator is not recommended as video support is buggy.  This will be resolved as Android Studio and the Android SDK mature.
+1. To run a sample app on an Android 5.0 or later device, plug the device into the computer and ensure that USB debugging is enabled.
```

Comment:
Android 5.0 is on passive support. Should we instead reference 6.0.1 or greater, since that's the minimum version on active support?